CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 
 * **[Fix]** Fix an issue where Push could not be started because `StartPush` method was stripped for a high managed stripping level.
 
+#### iOS/UWP
```

Comment:
I see an Android patch too in the PR which seems to change behavior since we move the replay method to start, I'm curious why it's not mentioned here.

Reply:
As I remember android part worked before and still works.

Reply:
So what was that change about then?

Reply:
That's change for iOS

Reply:
Are you saying that `Push.ReplayPushNotificationsIfWaiting();` was moved to `PostInitialize` in Android just to align it with the iOS implementation, but it worked fine when it was in the `Initialize` function?

Reply:
I am curious why android was changed as well but not mentioned in changelog. What is the rationale of that change.